packages/cli/src/commands/update.ts

```diff
@@ -0,0 +1,78 @@
+import fs from "fs";
+import path from "path";
+import fetch from "node-fetch";
+import inquirer from "inquirer";
+import { installDependencies } from "../utils/installDependencies.js";
+
+export async function updateCommand(projectPath: string) {
+  const packageJsonPath = path.join(projectPath, "package.json");
+
+  if (!fs.existsSync(packageJsonPath)) {
+    console.error("package.json not found in the current directory.");
+    return;
+  }
+
+  const packageJsonData = JSON.parse(fs.readFileSync(packageJsonPath, "utf8"));
+  const dependencies = packageJsonData.dependencies || {};
+  const devDependencies = packageJsonData.devDependencies || {};
+
+  const triggerPackages = Object.keys({ ...dependencies, ...devDependencies }).filter(
+    (packageName) => packageName.startsWith("@trigger.dev/")
+  );
+
+  if (triggerPackages.length === 0) {
+    console.log("No @trigger.dev/* packages found in package.json.");
+    return;
+  }
+
+  const newVersions = await Promise.all(
+    triggerPackages.map(async (packageName) => {
+      try {
+        const response = await fetch(`https://registry.npmjs.org/${packageName}`);
+        const data = await response.json();
+        const latestVersion = data["dist-tags"].latest;
+        return { packageName, latestVersion };
+      } catch (error) {
+        // @ts-ignore
+        console.error(`Error fetching version for ${packageName}: ${error.message}`);
+        return null;
+      }
+    })
+  );
+  const packagesToUpdate = newVersions.filter(
+    (entry) => entry !== null && entry.latestVersion !== dependencies[entry.packageName]
```

Comment:
It's most common to not have exact versions specified in the package.json file.

For example, after run the CLI init command you end up with this:

```json
"@trigger.dev/sdk": "^2.0.8",
"@trigger.dev/nextjs": "^2.0.8"
```

It's no good comparing the strings of the versions, even if you use something like semver comparison because you can't tell what's actually installed.

You need to do something like npm list to do this. That tells you which packages are actually installed on that machine.